native/explorer/src/datatypes.rs

```diff
@@ -4,7 +4,7 @@ use rustler::resource::ResourceArc;
 use rustler::{Atom, Encoder, Env, NifStruct, Term};
 use std::convert::TryInto;
 
-use crate::atoms;
+use crate::atoms::{self, infinity, nan, neg_infinity};
 use crate::atoms::{calendar, calendar_atom, day, hour, microsecond, minute, month, second, year};
```

Comment:
Do we want to combine these two use declarations?

Reply:
I changed that to one `use` declaration. Thanks!